README.md

```diff
@@ -107,11 +107,15 @@ After you clone the repository locally, there are several configuration steps re
 
 ##### CMS
 
-- Copy `server/conf/cms/secrets.sample.py` to `server/conf/cms/secrets.py`
+Add CMS settings and secrets:
 
-- To emulate a specific CMS project, copy https://github.com/TACC/Core-CMS-Resources/blob/main/__PROJECT_TO_EMULATE__/settings_custom.py to `server/conf/cms/settings_custom.py`
+```sh
+curl "https://cdn.jsdelivr.net/gh/TACC/Core-CMS@main/taccsite_cms/secrets.example.py" -o server/conf/cms/settings_custom.py
+cp server/conf/cms/secrets.sample.py server/conf/cms/secrets.py
+curl "https://cdn.jsdelivr.net/gh/TACC/Core-CMS@main/taccsite_cms/settings_local.example.py" -o server/conf/cms/settings_local.py
+```
```

Comment:
99% of use of the CMS in this repo by devs is just to have a homepage. I think a good goal would be to have a default config (the existing `secrets.sample.py`) to get up and running immediately, and if need be a dev who needs bespoke CMS settings can set them after the fact. Think we can figure something out?

Reply:
I had thought so, and am fine with minimal working secrets (as it was).

@taoteg reports devs adding CMS non-secret settings to CMS `secrets.py`. _Or maybe I misunderstood._ For fear that **is** happening:
1. I added support for `settings_custom.py` and `settings_local.py`.
2. To avoid `settings_custom.py` and `settings_local.py` being added as directories if they are absent, I added commands to create them.
3. To avoid needing to keep those files up-to-date, I `curl`ed them from TACC/Core-CMS.

Reply:
Personally I think keeping a working set of default settings committed makes more sense for local dev, and updates as needed can be committed to the repo.

How about this: We commit a set of local settings that are committed with dummy credentials that will be imported into the CMS settings on make start, and if a (untracked) `secrets.py` happens to exist, then that acts as an override

Reply:
I am looking for parity between env setups here. We should be using the same config files (settings, settings_custom, settings_local, secrets) in every env (dev, local, pprd, prod). 

By leaving out parts of the default config structure we are not training devs on the expected practice for how these files exist in pprd/prod and why certain settings exist in certain files.

Examples: 
- There is no value for `SILENCED_SYSTEM_CHECKS = ['captcha.recaptcha_test_key_error'] ` in the example settings. This error will creep up in the logging (annoying). [Supposedly this will go away when  `FP-1728: Remove once servers have RECAPTCHA_..._KEY settings` but so far has not for me.. maybe due to the old CMS image?]

This request is as much about consistency in workflows as it is about having good settings out of the box that use a current CMS image.

Reply:
Ah, the `SILENCED_SYSTEM_CHECKS` is in Core-CMS `settings_local.py`…. Another way to resolve that missing piece is add it to a committed and tracked `settings_custom.py` (which [@rstijerina proposed to us privately](https://tacc-team.slack.com/archives/G0143P1HQAK/p1733781148830159)).

Reply:
New solution available via `settings-default.py` (which Core-CMS supports in https://github.com/TACC/Core-CMS/pull/904.